frontend/public/components/storage-class.tsx

```diff
@@ -49,8 +54,6 @@ const StorageClassDetails: React.SFC<StorageClassDetailsProps> = ({obj}) => <Rea
       <dd>{obj.provisioner || '-'}</dd>
       <dt>Reclaim Policy</dt>
       <dd>{obj.reclaimPolicy || '-'}</dd>
-      <dt>Default Class</dt>
```

Comment:
Why would we remove this from the details page? I don't see where in the JIRA it says to remove it. The linked slides have a TODO but no clear guidance.

This will also conflict with #1339

Reply:
@julim@redhat.com